src/IceRpc.Transports.Quic/QuicTransportOptions.cs

```diff
@@ -9,9 +9,11 @@ public record class QuicTransportOptions
 {
     /// <summary>Gets or sets the idle timeout. This timeout is used to monitor the transport connection health. If no
     /// data is received within the idle timeout period, the transport connection is aborted.</summary>
-    /// <value>The idle timeout. Defaults to <see cref="TimeSpan.Zero" /> meaning that the underlying QUIC
-    /// implementation default will be used.</value>
-    public TimeSpan IdleTimeout { get; set; } = TimeSpan.Zero;
+    /// <value>The idle timeout. Defaults to 30 seconds. <see cref="TimeSpan.Zero" /> means "use the default value
+    /// provided by the underlying implementation".</value>
+    /// <remarks>The idle timeout is negotiated by QUIC during connection establishment. The actual idle timeout value
+    /// for a connection can be lower than the value you've set here.</remarks>
+    public TimeSpan IdleTimeout { get; set; } = TimeSpan.FromSeconds(30);
```

Comment:
Changed the default from Zero to 30 seconds - which is the msquic default (= what Zero meant). It's clearer to spell out the actual default, and it's highly desirable to use the same default in all IceRPC implementation.